### PR TITLE
Dashboard: add highlight of Jetpack blocks to the settings page

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

﻿Related: #23468

Let's promote that Jetpack includes blocks in the Jetpack settings, in the Jetpack dashboard.

image

Later on, we can add a toggle here that would act as a main toggle from this:
https://github.com/Automattic/jetpack/blob/a09c00ea3c1531eb15d5d8f58bc685dd7d916799/projects/plugins/jetpack/class.jetpack-gutenberg.php#L451-L474

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Jetpack > Settings > Writing
* You should see a card promoting Jetpack blocks, with a link to the block editor. Give it a try.
* Go to Plugins > Add New, and install / activate the Classic Editor plugin.
* Go back to Jetpack > Settings > Writing, the blocks card should be gone.

### CODE REVIEW

File: projects/plugins/jetpack/_inc/client/writing/composing.jsx [hunk 1]
```diff
@@ -152,6 +154,32 @@ export class Composing extends React.Component {
...
+								'Jetpack includes some blocks which can help you create your pages exactly the way you want them.',
+								'jetpack'
+							),
+							link: getRedirectUrl( 'jetpack-support-blocks' ),
+						} }
+					>
+						<p className="jp-settings-card__blocks-description">
+							{ __(
+								'Jetpack blocks give you the power to deliver quality content that hook website visitors without needing to hire a developer or learn a single line of code.',
+								'jetpack'
```
Comment (coder-karen) [inline projects/plugins/jetpack/_inc/client/writing/composing.jsx:172]:
```suggestion
								'Jetpack blocks give you the power to deliver quality content that hooks website visitors without needing to hire a developer or learn a single line of code.',
								'jetpack'
```
Same suggestion as in #23468.

**Review by coder-karen — COMMENTED** [review]
158655527-81b89c5e-98dc-4ca5-aba9-734cb94a2709

^ I wonder if the link can be in the same settings group as the initial information about Jetpack blocks just above? The separating line leads me to think these two sections aren't necessarily related.

**Comment by jeherve** [comment]
> ^ I wonder if the link can be in the same settings group as the initial information about Jetpack blocks just above? The separating line leads me to think these two sections aren't necessarily related.

I had considered that as well, but opted to follow the way we do things for other settings, like the Portfolio and Testimonial settings right below the block settings:

image

**Review by coder-karen — APPROVED** [review]

**Comment by jeherve** [comment]
Master merge to try to unblock things.

**Pull request merged**